src/games/strategy/triplea/delegate/StrategicBombingRaidBattle.java

```diff
@@ -203,7 +203,21 @@ public void fight(final IDelegateBridge bridge) {
     showBattle(bridge);
     final List<IExecutable> steps = new ArrayList<>();
     if (hasAA) {
-      steps.add(new FireAA());
+    	boolean foundGenericAA = false;
```

Comment:
It appears that is what is happening in the tests. If the suggested code really has the same functionality, and also works, then you can undo the new test addition, and all the tests will pass.